lib/model/yake.py

```diff
@@ -0,0 +1,51 @@
+from typing import Dict
+import io
+import urllib.request
+
+from lib.model.model import Model
+
+from lib import schemas
+
+from lib.logger import logger
+
+import yake
```

Comment:
This won't work. You cannot call the file `yake.py` and then `import yake` . The file must be called something different